src/StoreApi/Routes/Checkout.php

```diff
@@ -123,8 +123,10 @@ protected function get_route_response( \WP_REST_Request $request ) {
 	 * @return \WP_REST_Response
```

Comment:
The changes within this class ensure that totals reflect customer data posted via the form, so if the API updates don't run on the checkout itself, the correct taxes etc are still calculated.